tests/perl/PerlCriticBearTest.py

```diff
@@ -26,28 +27,15 @@
 """.splitlines(keepends=True)
 
 
-config_file = """
-severity  = 5
-# for signatures
-[-Subroutines::ProhibitSubroutinePrototypes]
-
-[TestingAndDebugging::RequireUseStrict]
-
-[TestingAndDebugging::RequireUseWarnings]
-""".splitlines(keepends=True)
-
+script_dir = os.path.dirname(os.path.realpath(__file__))
+conf_file = os.path.join(script_dir, "testfiles", ".perlcriticrc")
 
 PerlCriticBearTest = verify_local_bear(PerlCriticBear,
                                        valid_files=(good_file,),
                                        invalid_files=(bad_file,))
 
-
-with prepare_file(config_file,
-                  filename=None,
-                  force_linebreaks=True,
-                  create_tempfile=True) as (conf_lines, conf_file):
-    PerlCriticBearConfigTest = verify_local_bear(
-        PerlCriticBear,
-        valid_files=(bad_file, good_file),
-        invalid_files=(),
-        settings={"perlcritic_config": conf_file})
+PerlCriticBearConfigTest = verify_local_bear(
+    PerlCriticBear,
+    valid_files=(good_file,),
```

Comment:
You're the only one with blame information... I think that's because you moved everything to the new coala-bears repo a few weeks ago. Anyway I can find the original blame info?